scripts/src/main/resources/scripts/functions

```diff
@@ -19,27 +19,26 @@ DEVON_DOWNLOAD_DIR="${DEVON_HOME_DIR}/Downloads/devonfw-ide"
 # $2: optional exit code
 function doFail() {
   doEchoAttention ""
-  echo -e "ERROR: ${1}"
-  echo "We are sorry for the inconvenience. Please check the above errors, resolve them and try again."
+  doError "${1}\nWe are sorry for the inconvenience. Please check the above errors, resolve them and try again."
   if [ -n "${2}" ]
   then
-    echo "Exit code was ${2}"
+    doError "Exit code was ${2}"
     if [ "${2}" = "0" ]
     then
       exit 1
     fi
-    exit "${2}"
+    doError "${2}"
```

Comment:
this is what broke the build. The exit was replaced with logging causing an infinity loop.

Reply:
fixed.